mrtt-ui/src/components/CheckboxGroup/CheckboxGroup.js

```diff
@@ -62,54 +66,69 @@ const CheckboxGroup = forwardRef(
         getValueToReturn({
```

Comment:
I cant remember. It was important to return to the consumer for something. Maybe yup validation because an empty string in `otherValue` will come through as true if you test for that?

Reply:
I am realizing that I answered not your question, but some other question. . If we uncheck other, we leave the input in the front end..... BUT it might not get submitted..... I need to check this again to make sure it makes sense. I vaguely remember making a cut corner deliberately for this, but its been so long.

Reply:
unchecking other will clear the stakeholder benefits selects, but not the other value _in the ui_. Unchecking other will make sure the other value and the stakeholder benefits sent to the api are undefined/empty. It was a shortcut for timelines.  

In exploring this I found and fixed a bug! will add comment when I push it.